resources/public/api/conf/1.0/other_expenses_retrieve.yaml

```diff
@@ -0,0 +1,100 @@
+summary: Retrieve Other Expenses
+description: |
+  This endpoint enables you to retrieve other expenses.
+  A National Insurance number and tax year must be provided.
+  
+  ### Test data
+  <p>Scenario simulations using Gov-Test-Scenario headers is only available in the sandbox environment.</p>
+  <table>
+      <thead>
+          <tr>
+              <th>Header Value (Gov-Test-Scenario)</th>
+              <th>Scenario</th>
+          </tr>
+      </thead>
+      <tbody>
+          <tr>
+              <td><p>N/A - DEFAULT</p></td>
+              <td><p>Simulates success response.</p></td>
+          </tr>
+          <tr>
+              <td><p>NOT_FOUND</p></td>
+              <td><p>Simulates the scenario where no data is found.</p></td>
+          </tr>
+      </tbody>
+  </table>
+
+tags:
+  - Other Expenses
+
+parameters:
+  - $ref: './common/pathParameters.yaml#/components/parameters/nino'
+  - $ref: './common/pathParameters.yaml#/components/parameters/taxYear'
+  - $ref: './common/queryParameters.yaml#/traits/source'
```

Comment:
This can be removed

Reply:
Regarding that the spec is correct that is.. if it is then the success response will also need to be updated